tox.ini

```diff
@@ -74,11 +74,13 @@ setenv =
 deps =
   setuptools_rust  # This is work around for the bug of tox 3 (see #8606 for more details.)
   -r{toxinidir}/requirements-dev.txt
-  qiskit-aer
-  # Aer depends on Terra. We want to make sure pip satisfies that requirement from a local
-  # installation, not from PyPI. But Tox normally doesn't install the local installation until
-  # after `deps` is installed. So, instead, we tell pip to do the local installation at the same
-  # time as Aer. See https://github.com/Qiskit/qiskit-terra/pull/9477.
+  -r{toxinidir}/requirements-optional.txt
+  # Some of the optionals depend on Terra. We want to make sure pip satisfies
+  # that requirement from a local installation, not from PyPI, but Tox normally
+  # doesn't install the local installation until after `deps` is installed.
+  # So, instead, we tell pip to do the local installation at the same time.
+  # See https://github.com/Qiskit/qiskit-terra/pull/9477.
   .
```

Comment:
I changed this, but in retrospect I'm not convinced that I should have - I think maybe we should be ensuring that the docs build completely without _any_ optionals installed.  To take that to its completion, maybe we want to consider revising things such that Aer isn't needed to build the docs either (I'm not sure how prolifically it's used).

Reply:
I reverted this in 73e8e8157.